Library/Formula/librsvg.rb

```diff
@@ -3,9 +3,9 @@ class Librsvg < Formula
   homepage "https://live.gnome.org/LibRsvg"
   url "https://download.gnome.org/sources/librsvg/2.40/librsvg-2.40.10.tar.xz"
   sha256 "965c807438ce90b204e930ff80c92eba1606a2f6fd5ccfd09335c99896dd3479"
+  revision 1
 
   bottle do
-    sha256 "05e36f46fa553bef32ce12fac9b2aedb29ef0ada303a76417da3d7be42615d46" => :el_capitan
```

Comment:
You can leave it this time, but don't worry about the bottles. We're building for Mav/Yos/Cap these days so when bottles are pulled ML will vanish & the others will move down in the list one place. It doesn't leave "dead" bottles in place.